homeassistant/components/tile/device_tracker.py

```diff
@@ -65,52 +83,56 @@ class TileDeviceTracker(CoordinatorEntity, TrackerEntity):
 
     _attr_icon = DEFAULT_ICON
 
-    def __init__(self, entry, coordinator, tile):
+    def __init__(
+        self, entry: ConfigEntry, coordinator: DataUpdateCoordinator, tile: Tile
+    ) -> None:
         """Initialize."""
         super().__init__(coordinator)
 
-        self._attr_extra_state_attributes = {ATTR_ATTRIBUTION: DEFAULT_ATTRIBUTION}
+        self._attr_extra_state_attributes: MutableMapping[str, Any] = {
+            ATTR_ATTRIBUTION: DEFAULT_ATTRIBUTION
+        }
         self._attr_name = tile.name
         self._attr_unique_id = f"{entry.data[CONF_USERNAME]}_{tile.uuid}"
         self._entry = entry
         self._tile = tile
 
     @property
-    def available(self):
+    def available(self) -> bool:
         """Return if entity is available."""
         return super().available and not self._tile.dead
 
     @property
-    def location_accuracy(self):
+    def location_accuracy(self) -> int:
         """Return the location accuracy of the device.
 
         Value in meters.
         """
-        return self._tile.accuracy
+        return int(self._tile.accuracy)
 
     @property
     def latitude(self) -> float:
         """Return latitude value of the device."""
-        return self._tile.latitude
+        return cast(float, self._tile.latitude)
```

Comment:
According to the upstream library, this can return None, do we need to handle that?